sites/all/modules/reol_base/reol_base.module

```diff
@@ -1730,15 +1731,17 @@ function reol_base_opensearch_post_execute($request, $response, $raw_response) {
       switch (get_class($res)) {
         case 'TingClientObject':
           $res->on_quota = TRUE;
-          if (isset($raw_response->searchResponse->result->searchResult[$raw_index]->collection->object[0]->collection->record)) {
+          if (is_array($raw_response->searchResponse->result->searchResult[$raw_index]->collection->object) &&
```

Comment:
Are we absolutely sure that `$raw_response->searchResponse->result->searchResult` is an array here?

Reply:
jeps

Reply:
if it's not they I think the whole parsing in the tinc-client is f'ed